pgserver/query_modifier.go

```diff
@@ -17,6 +17,7 @@ func NewQueryRemover(regex string) QueryModifier {
 }
 
 var (
+	convertFloatToF64    = NewQueryReplacer(`(?i)\bFLOAT\b`, `DOUBLE`)
```

Comment:
Why?

Reply:
https://www.cockroachlabs.com/docs/stable/float

I see. But the result of `parser.Parse` should be unused in the code. Is it used anywhere?